client/components/commit_Items.jsx

```diff
@@ -0,0 +1,35 @@
+import React, {Component} from 'react';
+import CommitItem from './commit_item';
+
+export default class CommitItems extends Component {
+  componentWillMount(){
+      this.props.getCommitData();
+  }
+
+  showCommitData(){
+    // console.log(this.props.results.data);
```

Comment:
please remove solo-dev methods such as these comments and console logs before resubmitting pull request